demos/minimal/actors/sample_generator.py

```diff
@@ -15,54 +15,52 @@ class Generator(Actor):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.data = None
         self.name = "Generator"
-        self.frame_num = 0
+        self.frame_num = 0  # Initialize frame counter
+        # Generate xs (x-coordinates)
+        self.xs = np.linspace(-10, 10, 100)
 
     def __str__(self):
-        return f"Name: {self.name}, Data: {self.data}"
+        return f"Name: {self.name}"
 
     def setup(self):
-        """Generates an array that serves as an initial source of data.
-
-        Initial array is a 100 row, 5 column numpy matrix that contains
-        integers from 1-99, inclusive.
-        """
-
+        """Initial setup for Generator."""
```

Comment:
You still need to have a comment here about what data is being generated for users.